tests/php/Configuration/Parser/TaxonomyParserTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Tests\Configuration\Parser;
+
+use Bolt\Configuration\Parser\MenuParser;
+use Bolt\Configuration\Parser\TaxonomyParser;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Config\Exception\FileLocatorFileNotFoundException;
+use Symfony\Component\Yaml\Exception\ParseException;
+use Tightenco\Collect\Support\Collection;
+
+class TaxonomyParserTest extends TestCase
+{
+    public function testCanParse(): void
+    {
+        $taxonomyParser = new TaxonomyParser();
+        $config = $taxonomyParser->parse();
+
+        $this->assertInstanceOf(Collection::class, $config);
+    }
+
+    public function testBreakOnInvalidFileParse(): void
```

Comment:
duplicated test

Reply:
Except it tests it on a _different_ class.. Really think i should remove it?